### ISSUE TITLE
Add Chromatic To Project

### ISSUE THREAD

**Issue by madeleine-patience** [madeleine-patience/Considerate-Cat-Version-2-With-Next#52]
I think it would be cool for us to add chromatic to our project. We can use 5000 snapshots a month for free.  What are your thoughts on this @arthvadrr 

https://www.chromatic.com/features/visual-test

**Comment by arthvadrr** [comment]
I'd say as long as you know it won't incur any costs and you don't have to worry about that then sure. 5000 seems like more than enough on the free tier - but not if you have the burden of babysitting the usage numbers.

If we can bolt on a new package and learn from it then I think that's enough reason to do so and try it out. The added testing won't hurt either.

We might look at using testing-library, too.

**Comment by madeleine-patience** [comment]
Yeah sounds good, do you have any opinions on testing? Are you thinking e2e tests?

**Comment by arthvadrr** [comment]
I wasn't thinking E2E, but I'm open to it, haha.

If you're just looking for that experience, we could add something like Cypress. But it might be better to add things one step at a time, though. Maybe start with unit testing or Chromatic, and save E2E for last.
